requirements.txt

```diff
@@ -26,7 +26,7 @@ python-gnupg<0.4,>=0.3.7
 python-json-logger
 python-nmap<0.4,>=0.3.4
 pyyaml<4.0,>=3.11
-requests<3.0,>=2.7.0
+requests<3.0,>=2.7.0 [security]
```

Comment:
Thanks for the link. However i've started to dig into. 
Opposed to what is written there pip 7.1.2 **can not** parse `requests<3.0,>=2.7.0 [security]`
/cc @jfryman @Kami 
I'm looking to the issue.